pubsub/option.go

```diff
@@ -39,9 +42,10 @@ func GoogleJSON(projectID string, cred []byte) Option {
 }
 
 // GoogleJSON configures Publisher & Subscriber for Google Cloud Pub/Sub auth using JSON filename
-func GoogleJSONFile(projectID, filename string) Option {
+func GoogleJSONFile(projectID, filename, namespace string) Option {
```

Comment:
example & this func mismatch args

Reply:
updated

Reply:
`projectID, namespace string, cred []byte`
`projectID, filename, namespace string`
would be better with consistent args ordering

Reply:
updated
will use `projectID, namespace string, credential`